runner/scheduler.py

```diff
@@ -258,23 +258,29 @@ def setup_iterator(self, test_group):
             self.travel_to_dt(self.master_log.log[-1].timestamp)
 
         # Fast forward examples to the last action that was run
-        for run_id in test_actions_taken.keys():
-            example = tests[run_id]
-            actions_to_ff = test_actions_taken[run_id]
-            for _ in range(actions_to_ff):
-                action = example.step()
-                if isinstance(action, SendAndRegisterAction):
-                    self.register_callback(example)
+        # If the reset priority for the model is soft, then fast forward the tests, otherwise all tests in progress are discarded
```

Comment:
I got a bit confused by this comment, I'd rephrase it to "all messages from partial tests are discarded".